app/views/list/list.erb

```diff
@@ -44,33 +44,63 @@
+        <th></th>
         <th><%= t 'js.replies' %></th>
+        <th><%= t 'js.views' %></th>
         <th><%= t 'js.activity' %></th>
       </tr>
     </thead>
 
     <tbody>
       <meta itemprop='itemListOrder' content='http://schema.org/ItemListOrderDescending'>
       <% @list.topics.each_with_index do |t,i| %>
-        <tr>
-          <td itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
-            <meta itemprop='position' content='<%= i %>'>
-            <meta itemprop='url' content='<%= t.url %>'>
+        <tr class="topic-list-item">
+          <td class="main-link" itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
             <meta itemprop='name' content='<%= t.title %>'>
-            <a href='<%= t.url %>' class='title raw-link raw-topic-link'>
-              <span><%= t.title %></span>
-            </a>
-            <% if (!@category || @category.has_children?) && t.category %>
-              <div>
-                <a href='<%= t.category.url %>'><span class='category-name'><%= t.category.name %></span></a>
-              </div>
+            <meta itemprop='url' content='<%= t.url %>'>
+            <% if t.image_url.present? %>
+              <meta itemprop='image' content='<%= t.image_url %>'>
             <% end %>
+            <meta itemprop='position' content='<%= i + 1 %>'>
+            <span class="link-top-line">
+              <a href='<%= t.url %>' class='title raw-link raw-topic-link'><%= t.title %></a>
+            </span>
+            <div class="link-bottom-line">
+              <% if (!@category || @category.has_children?) && t.category %>
+                <a href='<%= t.category.url %>' class='badge-wrapper bullet'>
+                  <span class='badge-category-bg' style='background-color: #<%= t.category.color %>'></span>
+                  <span class='category-title'><%= t.category.name %></span>
+                </a>
+              <% end %>
+              <% if t.tags %>
+                <div class="discourse-tags">
+                  <% t.tags.each_with_index do |tag, index| %>
+                    <a href='<%= tag.full_url %>' class='discourse-tag'><%= tag.name %></a>
+                    <% if index < t.tags.size - 1 %>,&nbsp;<% end %>
+                  <% end %>
+                </div>
+              <% end %>
+            </div>
             <% if t.pinned_until && (t.pinned_until > Time.zone.now) && (t.pinned_globally || @list.category) && t.excerpt %>
               <p class='excerpt'>
                 <%= t.excerpt.html_safe %>
               </p>
             <% end %>
           </td>
+          <td class='posters'>
+            <% t.posters.each do |poster| %>
+              <% if poster.moreCount %>
+                <a class="posters-more-count"><% poster.moreCount %></a>
+              <% else %>
+                <a href="<%= Discourse.base_url %>/u/<%= poster.user.username %>" class="<%= poster.extraClasses %>">
```

Comment:
Let's maybe drop the topic list avatars, or put them inside an `if false`, until the topic view has them too.